selenium/che-selenium-test/src/test/resources/projects/plugins/DebuggerPlugin/php-tests/index.php

```diff
@@ -1,14 +1,4 @@
 <?php include 'lib.php';?>
-/*
```

Comment:
Why did you remove the license header? We have requested to set license headers above the all our source code in tests including resources.